src/GridFS/ReadableStream.php

```diff
@@ -48,13 +51,13 @@ class ReadableStream
     /** @var integer */
     private $chunkOffset = 0;
 
-    /** @var CursorInterface|null */
+    /** @var Cursor|null */
     private $chunksIterator;
 
     /** @var CollectionWrapper */
     private $collectionWrapper;
 
-    /** @var float|integer */
+    /** @var integer */
```

Comment:
Good catch. Since we already asserted that the file length was an integer and `$expectedLastChunkSize` is calculated as the difference of `length` and `numChunks` (integer-cast result of `ceil()`), this should always have an integer type.